src/AcmTable/AcmTable.tsx

```diff
@@ -770,6 +791,79 @@ export function AcmTable<T>(props: AcmTableProps<T>) {
                                 ))}
                             </ToolbarGroup>
                         )}
+                        {tableDropdown && (
```

Comment:
This is great, Feng! Just wondering if you could create some of these components before the return, so that they don't all have to be repeated with/without the tooltip?

Reply:
@KevinFCormier Yep I tried creating the components programmatically so we don't repeat the code but for some reason the created React.Element object doesn't render in the return. This is the error I get:
`Objects are not valid as a React child. If you meant to render a collection of children, use an array instead`

this is what I had before in the AcmTable function and I would just reference `dropdownResults` in the return.
```
// render dropdown
    const renderDropdown = (dropdownActions: IAcmTableDropdown) => {
        return (
            <ToolbarGroup variant="button-group">
                <ToolbarItem>
                    <Dropdown
                        toggle={
                            <DropdownToggle
                                isPrimary={!dropdownActions.isDisabled}
                                id={dropdownActions.id}
                                onToggle={() => setDropdownActionsOpen(!dropdownActionsOpen)}
                                toggleIndicator={CaretDownIcon}
                                isDisabled={dropdownActions.isDisabled}
                            >
                                {dropdownActions.toggleText}
                            </DropdownToggle>
                        }
                        isOpen={dropdownActionsOpen}
                        dropdownItems={dropdownActions.actions.map((action) => (
                            <DropdownItem
                                key={action.id}
                                isDisabled={action.isDisabled}
                                component={action.component}
                                styleChildren={false}
                            >
                                {action.children}
                            </DropdownItem>
                        ))}
                    />
                </ToolbarItem>
            </ToolbarGroup>
        )
    }

    // helper function to render the table dropdown actions with RBAC support
    const renderDropdownHelper = (dropdownActions: IAcmTableDropdown) => {
        if (dropdownActions.isDisabled) {
            return (
                <Tooltip
                    content={dropdownActions.disableText}
                    position="bottom"
                >
                    {renderDropdown(dropdownActions)}
                </Tooltip>
            )
        } else {
            return(renderDropdown(dropdownActions))
        }
    }

    const dropdownResults = renderDropdownHelper(dropdownActions!)
```
Not sure if you've seen this before?

Reply:
Hey Feng, I believe I got this working... tests pass and storybook is OK with or without the table dropdown.

Had to update a few variable names, but the main change was just to not try rendering when tableDropdown is undefined.
```
const dropdownResults = tableDropdown && renderDropdownHelper(tableDropdown)
```

Reply:
Thanks @KevinFCormier 
I also looked into it a bit more. It seems I needed to wrap the object in a `<Fragment>` to get it to render.
<img width="434" alt="image" src="https://user-images.githubusercontent.com/38960034/127560902-493daaf4-cddf-4c72-b206-d23a5171a64b.png">

Anyways I'll commit the new changes shortly.

Reply:
Hey Feng,

I didn't need to use <Fragment>... it should only be needed when you have multiple children.
```
                           </ToolbarGroup>
                        )}
                        {dropdownResults}
                        {Object.keys(selected).length > 0 && bulkActions.length > 0 && (
                            <ToolbarGroup variant="button-group">
```

But now I'm wondering if the <Tooltip> should actually be wrapping the <ToolbarGroup> or if it should just be around the <Dropdown>? Not sure if it makes a difference.

Reply:
@KevinFCormier You're right the `<Fragment>` is not needed so I removed it. I think it doesn't make much difference if we wrap the `<Tooltip>` around `<ToolbarGroup>` or `<Dropdown>`. At least I couldn't tell from testing.